### PR TITLE
Feature/hw dtserwthre 23 create transfer confirm details

### PR DESCRIPTION
**Pull request by aparthibanpaypal**
Updated Confirm Details Screen UI

### CODE REVIEW

File: Common/Sources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -231,7 +231,7 @@
 
 // Transfer Funds
 "mobileTransferFundsHeader" = "Transfer funds";
-"mobileConfirmationHeader" = "Confirm the details";
+"mobileConfirmationHeader" = "Confirm details";
```
Comment (jsaini-hw) [inline Common/Sources/Localization/en.lproj/Localizable.strings:234]: Leave this content as it is

File: Transfer/Sources/ScheduleTransferPresenter.swift [hunk 2]
```diff
@@ -61,6 +65,11 @@ final class ScheduleTransferPresenter {
 
     private func initializeSections() {
         sectionData.removeAll()
+
+        let confirmTrnasferSourceSection =
```
Comment (jsaini-hw) [inline Transfer/Sources/ScheduleTransferPresenter.swift:69]: typo in confirmTrnasferSourceSection

File: Common/Sources/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (aparthibanpaypal) [reply Common/Sources/Localization/en.lproj/Localizable.strings:234]: Sure will update.

File: Transfer/Sources/ScheduleTransferPresenter.swift (hunk 2 above)
Comment (aparthibanpaypal) [reply Transfer/Sources/ScheduleTransferPresenter.swift:69]: Sure will update.

**Review by jsaini-hw — APPROVED** [review]

**Review by nanbehera — APPROVED** [review]
verified and Approved.

**Review by pdamodarannair — APPROVED** [review]
Approved

File: Transfer/Sources/ScheduleTransferController.swift [hunk 3]
```diff
@@ -53,12 +54,14 @@ final class ScheduleTransferController: UITableViewController, UITextFieldDelega
         if let transferMethod = initializationData?[InitializationDataField.transferMethod]
             as? HyperwalletTransferMethod,
             let transfer = initializationData?[InitializationDataField.transfer] as? HyperwalletTransfer,
-            let didFxQuoteChange = initializationData?[InitializationDataField.didFxQuoteChange] as? Bool {
+            let didFxQuoteChange = initializationData?[InitializationDataField.didFxQuoteChange] as? Bool,
+            let transferSourceCellConfiguration = initializationData?[InitializationDataField.selectedTransferSource] as? TransferSourceCellConfiguration {
```
Comment (jsaini-hw) [inline Transfer/Sources/ScheduleTransferController.swift:58]: fix swift lint warning

**Review by jsaini-hw — APPROVED** [review]

**Pull request merged**